### ISSUE TITLE
Robo\Task\Remote\Ssh::run() loses 'message' and 'data' fields in case of successful execution.

### ISSUE THREAD

**Issue by vitalyspirin** [consolidation-org/Robo#369]
It has the following code in that function:
```php
       if (!$result->wasSuccessful()) {
            return $result;
        }

        return Result::success($this);
```
So before that point $result object has 'message' and 'data'. In case of successful execution though it recreates Result object with empty fields:
```php
    static function success(TaskInterface $task, $message = '', $data = [])
    {
        return new self($task, 0, $message, $data);
    }
```
It would be more convenient if both 'message' and 'data' would always be returned.

**Comment by greg-1-anderson** [comment]
This should, in fact, just be `$result = $this->executeCommand($command);`

I can make this change, but I'd like to wrap up #377 first.

**Comment by greg-1-anderson** [comment]
Fixed in ea00b0e.
